Main.js

```diff
@@ -1,28 +1,81 @@
 import React, { Component } from "react";
+import { StyleSheet, Text, TextInput, TouchableOpacity, View } from "react-native";
 import { StatusBar } from "expo-status-bar";
-import { StyleSheet, Text, TouchableOpacity, View } from "react-native";
+import * as Clipboard from 'expo-clipboard';
+import * as SplashScreen from 'expo-splash-screen';
 
-// Either a Sandbox or Production SDK URL
-const sdkUrl = "<url>";
+// Either a Local, Sandbox or Production SDK URL
+const sdkUrl = "https://app-sandbox.columnapi.com/tax-filing?params=eyJ0b2tlbiI6ImV5SmhiR2NpT2lKSVV6STFOaUo5LmV5Sm9iM04wWDJKaGJtdGZkWE5sY2w5cFpHVnVkR2xtYVdWeUlqb2lNemRtWTJKbE5qQXRNR1psWWkwME9EQmxMV0kxTUdRdE9ERTNaREpoWWpCbU1UUmhJaXdpYUc5emRGOWlZVzVyWDJsa0lqb3hMQ0psZUhBaU9qRTNNREF4TkRFd05EVjkuam16XzFDWVdoM19Ya3pjZk1zeE1XalZpcGNDLWVIckZ0dnpWRVRiVUotUSJ9&target=eyJpZCI6IlRZMjI6OlNjcmVlbnM6OkhvbWVTY3JlZW4iLCJwYXlsb2FkIjp7fX0%3D";
```

Comment:
This is cool but could potentially get confusing if multiple partners are using this user at once

Reply:
This one i left untouched, i changed the url only.

Reply:
removed